### DISCUSSION TITLE
The first batch of rewards in our updated incentive program is just allocated

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#328] [Announcements]
As you may already know, all the liquidity providers to Tinyman’s ASASTATS-ALGO liquidity pool are qualified for the weekly rewards. For the first 38 weeks, we had been allocating rewards based on the initial setup of our [LP rewards incentive program](https://www.reddit.com/r/asastats/comments/slm2b4/asa_stats_is_officially_rolling_out_proposed_lp/). 

Starting today, and for week 39 of our incentive program, there's no cap on rewards for the liquidity providers! All the providers are qualified based on the same rule: for every [LP token](https://algoexplorer.io/asset/552659276) you hold in your account between two Sundays at 00:00 UTC, you will get 2.25 ASASTATS in the allocations that take place either on Sunday or Monday. Such a coefficient represents a 25% decrease in rewards for the smaller contributors and a 650% increase for the others.

[Provide liquidity on Tinyman](https://app.tinyman.org/#/pool/add-liquidity?asset_1=393537671&asset_2=0)

As always, everything is subject to discussion in ASA Stats so there's no guarantee or prediction for how many weeks this week's coefficient of **2.25** will be active - every eventual change will be announced here in this thread.

Increased APR for our LP providers would probably cause increased rewards on the very next [Cometa's](https://app.cometa.farm/stake) ASASTATS staking program that should take place in the week that starts on Monday 21st of November.

**Comment by delaw97** [comment]
👏 been with Asa stats since the early days when I received airdrops for being one of the first twitter followers... Good job

**Comment by Ansuz4u** [comment]
Why only Tinyman LP?  Why not AlgoFi also???

**Reply by ipaleka** [reply]
> Why only Tinyman LP? Why not AlgoFi also???

The primary outcome from our discussions in Discord and [here](https://github.com/asastats/channel/discussions/62) is that we shouldn't scatter our liquidity across many providers and pools until the liquidity increases. Once when, among other things, this incentive provides a result in that regard then we'll extend our program to other LPs.

**Comment by ipaleka** [comment]
For at least the next two weeks, the allocation of rewards in our own incentive program will take place on Monday evening CET.

We have discussed previously that Sunday isn't really suitable, so this change would probably stay.

**Comment by moloch10** [comment]
How does the launch of TM2 affect this program? Things like migration and cycles, etc..

**Reply by ipaleka** [reply]
I guess you've just started a discussion about that. :)

If we decide to migrate, from a technical perspective we'll surely try to send rewards no matter when in a week someone migrates.

**Reply by ipaleka** [reply]
By GamingToups in [Discord](https://discord.com/channels/906917846754418770/914540797054025788/1060600759089115156):
- - - - -
Tinyman V2 LP pools seemly launched, Main LP should be moved over asap to prevent issues with ratios due to buys or sells into low LP
Personal I've pulled all my V1 LPs instead of using the migration tool in expectation of volatility during move over.  Can readd after things have settled, since these things are rarely painless

**Reply by ipaleka** [reply]
By RandomTask in [Discord](https://discord.com/channels/906917846754418770/910524302531649566/1060603271653359616):
- - - - -
Guys, timyman v2 is live and we should all migrateliquidity to v2 protocol.

**Reply by ipaleka** [reply]
As our prominent members GamingToups and RandomTask suggested migrating, I'm about to migrate the devpool's funds in the next hour if nobody complains.

**Reply by moloch10** [reply]
Sounds good, I'll start the migration. Thanks

**Reply by AlgoRhythMatic** [reply]
Starting my migration now as well…

**Reply by AlgoRhythMatic** [reply]
Looks like the resulting v2 token count is roughly 8% higher than v1.  Should we retain the 2.25x multiplier?

**Reply by ipaleka** [reply]
> Looks like the resulting v2 token count is roughly 8% higher than v1. Should we retain the 2.25x multiplier?

I guess keeping it should be our choice. What is more important for now is to adjust @dragmz script for both pools in parallel, dunno exactly how we'll do that, but we'll do it. :)

**Comment by ipaleka** [comment]
An excerpt from the Transparency report for January 2023:

- - - - -
For participation in the ASA Stats LP rewards incentive program, from [Sun, 13 Feb 2022 16:40:19 UTC](https://algoexplorer.io/tx/YU3WETNCTLBXTIUGFIUTZES34TUOUV3L7GFJA34XHDKYWUQY6C7A) to [Mon, 23 Jan 2023 19:04:05 UTC](https://algoexplorer.io/tx/NG5J5TBX4Q7Q2TN6YMMYUFBMPMQEKIA3JKY77OGZ2XYC3IENJ54Q) the devpool received a total of 9,053,033 ASASTATS in 51 transactions from the airpool. As the rwdpool is assigned to be the next source of funds for the ASA Stats LP rewards incentive program – while it is planned that the airpool will continue to be the sender in the incentive program - the devpool sent 9,053,033 ASASTATS on [Mon, 23 Jan 2023 19:39:22 UTC](https://algoexplorer.io/tx/IHTX37ZQRWDAE6HSQSWLDCE65Q7YBCIXKTKQV7XKDTX5JO5T3UFQ) to the airpool on behalf of the rwdpool.
- - - - -
